docs/getting_started/tutorial-exogenous-processes.ipynb

```diff
@@ -0,0 +1,994 @@
+{
```

Comment:
I do not understand your first point.

I like the idea, but not for the first example because it is already complex enough. And I would keep the health example as the first, because it is the simplest. But, we could do something similar to the weather. If there is a storm in the current period, it is more likely for the following as well. At the same time we keep a low probability for storms after sunny weather.